projects/Mallard/src/screens/article-screen.tsx

```diff
@@ -63,23 +64,45 @@ const ArticleScreenWithProps = ({
         navigation: NavigationScreenProp<{}, ArticleNavigationProps>
     }) => {
     const pillar = getAppearancePillar(articleNavigator.appearance)
+    const firstUpdate = useRef(true)
```

Comment:
This doesn't seem to be used anywhere?

Reply:
it's right below it? the purpose is to blank the screen after rotations so you dont see all the nasty relayouts

Reply:
Oops sorry - I've no idea how I missed that?! Sorry!